app/services/proofing/lexis_nexis/verification_error_parser.rb

```diff
@@ -50,7 +50,7 @@ def parse_product_error_messages
           # don't log PhoneFinder reflected PII
           product.delete('ParameterDetails') if product['ProductType'] == 'PhoneFinder'
```

Comment:
should we update this check to be ExecutedStepName as well?
```suggestion
          product.delete('ParameterDetails') if product['ExecutedStepName'] == 'PhoneFinder'
```

Reply:
Frustrating, but actually no. We want to delete this data from all PhoneFinder products.